<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>fermentationEditor</class>
 <widget class="QDialog" name="fermentationEditor">
  <property name="windowTitle">
   <string>Fermentation Editor</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_9">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <property name="spacing">
      <number>2</number>
     </property>
     <item>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="1">
        <widget class="QLabel" name="label_name">
         <property name="toolTip">
          <string>Required</string>
         </property>
         <property name="text">
          <string>Name</string>
         </property>
         <property name="buddy">
          <cstring>lineEdit_name</cstring>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="SmartLineEdit" name="lineEdit_name">
         <property name="toolTip">
          <string>Name</string>
         </property>
        </widget>
       </item>

       <item row="2" column="1">
        <widget class="QLabel" name="label_description">
         <property name="toolTip">
          <string>Required</string>
         </property>
         <property name="text">
          <string>Description</string>
         </property>
         <property name="buddy">
          <cstring>textEdit_description</cstring>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QTextEdit" name="textEdit_description">
         <property name="toolTip">
          <string>Description</string>
         </property>
        </widget>
       </item>

       <item row="4" column="1">
        <widget class="SmartLabel" name="label_preFermentationSize">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="contextMenuPolicy">
          <enum>Qt::CustomContextMenu</enum>
         </property>
         <property name="text">
          <string>Pre-Fermentation Size</string>
         </property>
         <property name="buddy">
          <cstring>lineEdit_preFermentationSize</cstring>
         </property>
        </widget>
       </item>
       <item row="4" column="2">
        <widget class="SmartLineEdit" name="lineEdit_preFermentationSize">
         <property name="toolTip">
          <string>Pre-Fermentation Size</string>
         </property>
        </widget>
       </item>

       <item row="6" column="1">
        <widget class="SmartLabel" name="label_fermentationTime">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="contextMenuPolicy">
          <enum>Qt::CustomContextMenu</enum>
         </property>
         <property name="text">
          <string>Fermentation Time</string>
         </property>
         <property name="buddy">
          <cstring>lineEdit_fermentationTime</cstring>
         </property>
        </widget>
       </item>
       <item row="6" column="2">
        <widget class="SmartLineEdit" name="lineEdit_fermentationTime">
         <property name="toolTip">
          <string>Fermentation Time</string>
         </property>
        </widget>
       </item>

       <item row="8" column="1">
        <widget class="SmartLabel" name="label_notes">
         <property name="text">
          <string>Notes</string>
         </property>
        </widget>
       </item>
       <item row="8" column="2">
        <widget class="QTextEdit" name="textEdit_notes"/>
       </item>

       <item row="10" column="2">
        <widget class="QDialogButtonBox" name="buttonBox">
        <property name="orientation">
        <enum>Qt::Horizontal</enum>
        </property>
        <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
        </property>
        </widget>
       </item>

      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SmartLabel</class>
   <extends>QLabel</extends>
   <header>widgets/SmartLabel.h</header>
   <slots>
    <signal>changedSystemOfMeasurementOrScale(PreviousScaleInfo)</signal>
    <slot>popContextMenu(QPoint)</slot>
   </slots>
  </customwidget>
  <customwidget>
   <class>SmartLineEdit</class>
   <extends>QLineEdit</extends>
   <header>widgets/SmartLineEdit.h</header>
   <slots>
    <signal>textModified()</signal>
    <slot>lineChanged()</slot>
    <slot>lineChanged(PreviousScaleInfo)</slot>
   </slots>
  </customwidget>
 </customwidgets>
 <resources/>
</ui>
